lib/cucumber/core/test/result.rb

```diff
@@ -177,6 +177,24 @@ def ok?(be_strict = false)
           end
         end
 
+        class Ambiguous < Raisable
+          include Result.query_methods :ambiguous
+
+          def describe_to(visitor, *args)
+            visitor.failed(*args)
+            visitor.duration(duration, *args)
+            self
+          end
+
+          def to_s
+            "F"
+          end
+
+          def ok?(be_strict = false)
+            !be_strict
```

Comment:
Shouldn't this be `false` so that the exit code becomes non-zero when an ambiguous step is found?